pytest_fixtures/core/contenthosts.py

```diff
@@ -41,12 +41,22 @@ def host_conf(request):
     return conf
 
 
+def run_post_deployments(request, host):
+    """Execute post deploy"""
+    params = {}
+    if hasattr(request, 'param'):
+        params = request.param
+    if 'fips' in params.get('rhel_version'):
+        Broker().execute(workflow='enable-fips', target_host=host.name)
```

Comment:
Did you consider putting this logic into `ContentHost.setup` method? If we would go that route, we would need to pass `params['rhel_version']` (as a metadata) to broker through the `host_conf` method, so we can read it later on in the `setup` method. This is something we already do with `blank=True`.

https://github.com/ogajduse/robottelo/blob/8fff2ede1a86937f0dd6f4275b3b603de39c8c3b/pytest_fixtures/component/provision_pxe.py#L231

Reply:
Did not consider that for sure, but It was a perfect suggestion.

I choose this route to keep the content host procedures native to content host fixtures in this module for easy understanding and debug.

But anyways now we had to close the PR :(

Reply:
@ogajduse We are again allowed to go this path after some brainstorming in the Lab team. I have addressed your comment and its subject to verification post `enable-fips` WF merged in Tower.